lib/core/src/obf.cpp

```diff
@@ -78,6 +78,8 @@ int cipherBlockChaining = 0;
    HASH_TYPE_MD5 and HASH_TYPE_SHA1 */
 static int defaultHashType = HASH_TYPE_MD5;
 
+constexpr auto max_path_length{MAX_NAME_LEN + 10};
```

Comment:
Just curious ...

Any ideas as to why 10 is added?
Can 10 be replaced with a different value?

Reply:
Not sure why there is a +10… I just know that it shows up a lot. It's part of the reason I decided to give it a name here, at the very least.

Reply:
i really don't want this to be the answer... but '@tempZone' happens to be 10...

Reply:
well, okay, not 10, but ... yeah, i don't know.

Reply:
If we don't know, that's fine. It has an appropriate name now which is better than nothing.

Carry on.